wntr/epanet/io.py

```diff
@@ -667,6 +668,10 @@ def _write_tanks(self, f, wn, version=2.2):
                  'curve': '',
                  'overflow': '',
                  'com': ';'}
+            if tank.init_level < tank.min_level:
```

Comment:
Not actually sure what this batch of code is here for...

Reply:
@dbhart can you respond to this question?

Reply:
This logic should not be in io.py